### ISSUE TITLE
Bug: agent-install.sh should check podman networkbackend is netavark

### ISSUE THREAD

**Issue by johnwalicki** [open-horizon/anax#3268]
### Describe the bug.

On systems (oftentimes RHEL 8.6 / Fedora 36) that have Podman installed instead of Docker, the anax agent requires the network backend to be configured to use `netavark` instead of `cni` so that the agent can set up networking scenarios such as dependent services between containers.

The `agent-install.sh` checks if Podman 4.x is installed but should also check that `networkbackend` is configured to use `netavark`
Warn the user to switch the network backend.  Not certain if this should be a fatal error because anax will work with singleton containers.

If it is set for cni you have to install the netavark package and its dependencies, and then switch the networking from cni to netavark and then reboot.

These instructions provide details for the user.
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/building_running_and_managing_containers/index#proc_switching-the-network-stack-from-netavark-to-cni_assembly_setting-container-network-modes

We should make the agent-install.sh more robust. 
`podman info | grep -i networkbackend | cut -f4 -d' '`  then check that the value equals netavark

Its too complicated for the agent-install.sh script to do this.  `podman system reset` could be quite destructive.

Here's a simple bash script
```sh
#!/bin/bash
echo $(podman info | grep -i networkbackend | cut -f4 -d' ')
networkbackend=$(podman info | grep -i networkbackend | cut -f4 -d' ')
if [ $networkbackend != "netavark" ]; then
    echo "Error - change the podman network backend to netavark"
    # log_fatal 2 "Change the podman network backend to netavark"
fi
```

### Operating Environment

RHEL86 or Fedora

**Comment by dlarson04** [comment]
I am thinking to just leave the agent-install as-is. 

A user may just be installing the horizon-cli so they wouldn't need the netavark networking to do that... 

I think beefing up the horizon documentation around RHEL8.6 and talk about the netavark networking to run services is the best way to handle podman and not do anything with agent-install

We would just pointing to the RH documentation on how to switch the networking backend

**Comment by johnwalicki** [comment]
Ok - I'll track the update to `open-horizon.github.io/docs/installing/adding_devices.md` here because it seems redundant to open another ticket in the Docs repo.

https://open-horizon.github.io/docs/installing/adding_devices.html#rhel8

Related to RHEL 8.6 and Fedora 36 support tickets:
- https://github.com/open-horizon/open-horizon.github.io/issues/260
- https://github.com/open-horizon/anax/issues/3228
- https://github.com/open-horizon/open-horizon.github.io/issues/266
- https://github.com/open-horizon/anax/issues/3257

**Comment by johnwalicki** [comment]
We should also document netavark steps in `open-horizon.github.io/docs/installing/advanced_man_install.md`

**Comment by johnwalicki** [comment]
Closed via #3337
